pages/style-guide/prose.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Prose Style Guide
+---
```

Comment:
@scottgonzalez Excuse me, what do you mean when you say that this should be "JSON"?